projects/qpid-proton/project.yaml

```diff
@@ -3,3 +3,7 @@ primary_contact: "jross@apache.org"
 auto_ccs:
  - "security@apache.org"
  - "jdanek@redhat.com"
+sanitizers:
+ - address
+ - memory
```

Comment:
Assuming you've tried running with msan and there are no instant problems 
(msan needs to instrument *everything* and I can't tell from the build.sh if this is the case here)

Reply:
no instant problems

Reply:
> msan needs to instrument everything and I can't tell from the build.sh if this is the case here

I haven't done anything with the sanitizers a while now, so I do not exactly remember the background here... i would get a warning at startup if msan was not linked right, yes? So since I do not see such warning in the logs, this should be ok.